randovania/games/dread/logic_database/Ferenia.txt

```diff
@@ -1443,8 +1445,10 @@ Extra - asset_id: collision_camera_020
   > Tunnel to Energy Recharge Station (Gate)
       All of the following:
           Morph Ball and Screw Attack
-          # In case of entering with just 2 PBs, pickup can be third PB or (cross) bombs
-          Bomb or Cross Bomb or Power Bombs ≥ 3
+          Any of the following:
+              # In case of entering with just 2 PBs, pickup can be third PB or (cross) bombs
+              Bomb or Cross Bomb
+              Power Bombs ≥ 3 and Lay Power Bomb
```

Comment:
will this not make it think it needs 4 power bombs to go through? likewise with the above entry thinking it needs 3. No harm in just making it Main Power Bomb since Morph is already accounted for.

Reply:
Ammo isn't accumulated in the same way that damage is. It's just a capacity check. And I think it's fine either way to use the template unless there's a reason to only use the individual component.